minerl/Malmo/Minecraft/run/config/forge.cfg

```diff
@@ -14,7 +14,7 @@ D:java8Reminder=0.0
 B:disableStairSlabCulling=false
 
 # Enable forge to queue all chunk updates to the Chunk Update thread. May increase FPS significantly, but may also cause weird rendering lag. Not recommended for computers without a significant number of cores available.
-B:alwaysSetupTerrainOffThread=false
+B:alwaysSetupTerrainOffThread=true
```

Comment:
what does this do?

Reply:
This uses multi-threaded terrain / chunk loading - reverting